Backend/README.md

```diff
@@ -100,14 +101,14 @@ To speed up the compilation times, we use 6 parallel jobs by default. If you hav
 
 #### Running external services
 
-To run the project, we'd first need to run some services. These are provided via docker images, that are built in Nix and run via [arion].
+To run the project, we'd first need to run some services. These are provided via [services-flake].
 
-For running the database, redis, passetto and kafka run this command:
+For running the database, redis, passetto, osrm-server and kafka run this command:
 
 ```sh
-# NOTE: You must run this from inside nix develop shell.
-# The `-d` option will run the containers in background. Remove `-d` if you want to run them in foreground.
-, run-svc -d
+# Note: This will create a `data` directory in the root of the project, it will persist the data
+# on restart. If you want a fresh start, you can delete the entire `data` directory and re-run:
+nix run .#services
```

Comment:
I think `, run-svc` (which people already use) should run this.  In scripts.nix.

And arion stuff can be started using `, run-svc-docker` or something.

Reply:
Makes sense. Although `, run-svc-docker` may not be needed because for monitoring and pgadmin they do have separate commands.